components/automate-ui/src/app/entities/clients/client.effects.ts

```diff
@@ -38,128 +38,126 @@ export class ClientEffects {
-
-  @Effect()
-  deleteClientSuccess$ = this.actions$.pipe(
-    ofType(ClientActionTypes.DELETE_SUCCESS),
-    map(({ payload: { name } }: DeleteClientSuccess) => {
-      return new CreateNotification({
+  createClient$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ClientActionTypes.CREATE),
+      mergeMap((action: CreateClient) =>
+        this.requests.createClient(action.payload).pipe(
+          map((resp: CreateClientSuccessPayload) => new CreateClientSuccess(resp)),
+          catchError(
+            (error: HttpErrorResponse) => observableOf(new CreateClientFailure(error)))))));
+
+  createClientSuccess$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ClientActionTypes.CREATE_SUCCESS),
+      map(({ payload: { name } }: CreateClientSuccess) => new CreateNotification({
         type: Type.info,
-        message: `Successfully Deleted Client - ${name}.`
-      });
-    }));
-
-  @Effect()
-  deleteClientFailure$ = this.actions$.pipe(
-    ofType(ClientActionTypes.DELETE_FAILURE),
-    map(({ payload: { error } }: DeleteClientFailure) => {
-      const msg = error.error;
-      return new CreateNotification({
+        message: `Created client ${name}`
+    }))));
+
+  createClientFailure$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ClientActionTypes.CREATE_FAILURE),
+      filter(({ payload }: CreateClientFailure) => payload.status !== HttpStatus.CONFLICT),
+      map(({ payload }: CreateClientFailure) => new CreateNotification({
         type: Type.error,
-        message: `Could not delete client: ${msg || error}`
-      });
-    }));
+        message: `Could not create client: ${payload.error.error || payload}`
+    }))));
+
+  deleteClient$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ClientActionTypes.DELETE),
+      mergeMap(({ payload: { server_id, org_id, name } }: DeleteClient) =>
+        this.requests.deleteClient(server_id, org_id, name).pipe(
+          map(() => new DeleteClientSuccess({ name })),
+          catchError((error: HttpErrorResponse) =>
+            observableOf(new DeleteClientFailure(error)))))));
+
+  deleteClientSuccess$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ClientActionTypes.DELETE_SUCCESS),
+      map(({ payload: { name } }: DeleteClientSuccess) => {
+        return new CreateNotification({
+          type: Type.info,
+          message: `Successfully Deleted Client - ${name}.`
+        });
+    })));
+
+  deleteClientFailure$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ClientActionTypes.DELETE_FAILURE),
+      map(({ payload: { error } }: DeleteClientFailure) => {
+        const msg = error.error;
+        return new CreateNotification({
+          type: Type.error,
+          message: `Could not delete client: ${msg || error}`
+        });
+    })));
 
-  @Effect()
-  resetKeyClient$ = this.actions$.pipe(
+  resetKeyClient$ = createEffect(() => this.actions$.pipe(
```

Comment:
[nitpick] Of the 364 effects, there are just 14 (like this one) that do not have a line break after the `=>`. 😉